plugins/importexport/pubmed/locale/es_ES/locale.xml

```diff
@@ -12,22 +12,22 @@
   -->
  
 <locale name="es_ES" full_name="Español (España)">
-	<message key="plugins.importexport.pubmed.displayName">Plugin de exportación de XML de PubMed</message>
+	<message key="plugins.importexport.pubmed.displayName">Módulo de exportación de XML de PubMed</message>
 	<message key="plugins.importexport.pubmed.description">Exportación de la metadata de artículos usando XML de PubMed para Indexación en MEDLINE.</message>
 	
-	<message key="plugins.importexport.pubmed.export">Exportar Datos</message>
-	<message key="plugins.importexport.pubmed.export.issues">Exportar Números</message>
-	<message key="plugins.importexport.pubmed.export.selectIssue">Seleccione un Número para exportar.</message>
-	<message key="plugins.importexport.pubmed.export.articles">Exportar Artículos</message>
-	<message key="plugins.importexport.pubmed.export.selectArticle">Seleccione Artículos para exportar.</message>
+	<message key="plugins.importexport.pubmed.export">Exportar datos</message>
+	<message key="plugins.importexport.pubmed.export.issues">Exportar números</message>
+	<message key="plugins.importexport.pubmed.export.selectIssue">Seleccione unnNúmero para exportar.</message>
```

Comment:
I think there is a typo here!

Reply:
`unnNúmero` should be `un Número`

Reply:
That's what I thought, thanks -- could you update the PR?

Reply:
Done @asmecher

Reply:
Thanks, @kant!